testscript/testscript_test.go

```diff
@@ -121,6 +123,30 @@ func TestScripts(t *testing.T) {
 	// TODO check that the temp directory has been removed.
 }
 
+// TestTestwork tests that using the flag -testwork will make sure the work dir isn't removed
+// after the test is done. It uses an empty testscript file that doesn't do anything.
+func TestTestwork(t *testing.T) {
+	out, err := exec.Command("go", "test", ".", "-testwork", "-v", "-run", "^TestScripts/nothing$").CombinedOutput()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	re := regexp.MustCompile(`\s+WORK=(\S+)`)
```

Comment:
We probably want to match an entire line?

Reply:
I'm not sure it adds anything since it only is a timestamp before the match.

I should however update to use FindAllStringSubmatch to make sure that the test output doesn't contain more than one line.

Reply:
Fixed